packages/landing/src/debug.ts

```diff
@@ -92,9 +101,29 @@ export function addDebugLayer(bm: Basemaps): void {
     };
     debugEl.appendChild(osmButton.container);
 
+    const mouseEl = kv(`Mouse (${bm.config.projection.toEpsgString()})`, '');
+    mouseEl.container.className = 'debug__mouse';
+    const mouseWgs84El = kv(`Mouse (${Epsg.Wgs84.toEpsgString()})`, '');
+    mouseWgs84El.container.className = 'debug__mouse';
+
+    debugEl.appendChild(mouseEl.container);
+    debugEl.appendChild(mouseWgs84El.container);
+
+    bm.map.getViewport().addEventListener('mousemove', (e) => {
+        const px = bm.map.getEventPixel(e);
+        const coord = bm.map.getCoordinateFromPixel(px);
+
+        const wgs84 = proj
+            .transform(coord, bm.config.projection.toEpsgString(), Epsg.Wgs84.toEpsgString())
+            .map(round7)
+            .join(', ');
+        mouseWgs84El.value.innerHTML = wgs84;
+        mouseEl.value.innerHTML = coord.map(round3).join(', ');
```

Comment:
`textContent` would be a lot faster than `innerHTML`

Reply:
my simple benchmark shows this not to be the case

![image](https://user-images.githubusercontent.com/1082761/85637865-762be200-b6d8-11ea-86dc-4d6d662c6446.png)